### ISSUE TITLE
Unable to select anything other than Passive AI in Solo maps.

### ISSUE THREAD

**Issue by Zombie-Ryushu** [Wargus/war1gus#165]
Unable to select anything other than Passive AI in Solo maps. Te only option is: wc1-passive in Stratagus 2.4.3.

**Comment by Zombie-Ryushu** [comment]
Also they seem to not want to build their own Cobbled Road.

**Comment by Zombie-Ryushu** [comment]
Scratch that, Cobbled Road does not work at all!

**Comment by Zombie-Ryushu** [comment]
/usr/lib64/liblua.so.5.1[0x3e1f016cb7]
/usr/lib64/liblua.so.5.1[0x3e1f00c9dd]
/usr/lib64/liblua.so.5.1[0x3e1f00bcfb]
/usr/lib64/liblua.so.5.1[0x3e1f00cb4b]
/usr/lib64/liblua.so.5.1(lua_pcall+0x48)[0x3e1f008838]
stratagus(_ZN11LuaCallback3runEi+0x1b)[0x50140b]
stratagus(_ZN17LuaActionListener6actionERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEE+0x80)[0x541720]
stratagus(_ZN3gcn6Widget14generateActionEv+0x2d)[0x4b30fd]
stratagus(_ZN3gcn6Widget18_mouseInputMessageERKNS_10MouseInputE+0x3d6)[0x4b2a66]
stratagus(_ZN3gcn9Container18_mouseInputMessageERKNS_10MouseInputE+0x10e)[0x4b474e]
stratagus(_ZN3gcn3Gui5logicEv+0x127)[0x4ad627]
stratagus(_Z18WaitEventsOneFramev+0x159)[0x586129]
stratagus(_ZN10MenuScreen3runEb+0x96)[0x547586]
stratagus[0x5a96a0]
/usr/lib64/liblua.so.5.1[0x3e1f00c598]
/usr/lib64/liblua.so.5.1[0x3e1f016d4a]
/usr/lib64/liblua.so.5.1[0x3e1f00c9dd]
/usr/lib64/liblua.so.5.1[0x3e1f00bcfb]
/usr/lib64/liblua.so.5.1[0x3e1f00cb4b]
/usr/lib64/liblua.so.5.1(lua_pcall+0x48)[0x3e1f008838]
stratagus(_ZN11LuaCallback3runEi+0x1b)[0x50140b]
stratagus(_ZN17LuaActionListener6actionERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEE+0x80)[0x541720]
stratagus(_ZN3gcn6Widget14generateActionEv+0x2d)[0x4b30fd]
stratagus(_ZN3gcn6Widget18_mouseInputMessageERKNS_10MouseInputE+0x3d6)[0x4b2a66]
stratagus(_ZN3gcn9Container18_mouseInputMessageERKNS_10MouseInputE+0x10e)[0x4b474e]
stratagus(_ZN3gcn3Gui5logicEv+0x127)[0x4ad627]
stratagus(_Z18WaitEventsOneFramev+0x159)[0x586129]
stratagus(_ZN10MenuScreen3runEb+0x96)[0x547586]
stratagus[0x5a96a0]
/usr/lib64/liblua.so.5.1[0x3e1f00c598]
/usr/lib64/liblua.so.5.1[0x3e1f016d4a]
/usr/lib64/liblua.so.5.1[0x3e1f00c9dd]
/usr/lib64/liblua.so.5.1[0x3e1f00bcfb]
/usr/lib64/liblua.so.5.1[0x3e1f00cb4b]
/usr/lib64/liblua.so.5.1(lua_pcall+0x48)[0x3e1f008838]

**Comment by timfel** [comment]
Could you elaborate? What are you doing, which tool are you using, where are you expecting what? How is the stack trace related to your issue description? What were you doing when that occurred?


The AI never builds its own roads. There is some code in the default AI script that generates roads in certain intervals. The passive AI doesn't build anything, really.

**Comment by Zombie-Ryushu** [comment]
I need to close this issue and re-open a new one with a Title of the Installer installing a Corrupt instance. Because that's what actually happened. I found the mistakes the Installer made, and corrected them. The crash traces were from file installation errors caused by the installer. I fixed it by hand, but the Installer needs an upgrade mode to upgrade from Prior installations of WC1 and WC2.
